sds_data_manager/lambda_code/SDSCode/dependency_config.csv

```diff
@@ -110,6 +110,10 @@ mag, l1b, burst-magi, mag, l1c, norm-magi, HARD, DOWNSTREAM
 
 swapi, l0, raw, swapi, l1, all, HARD, DOWNSTREAM
 
+# <---- TEST SWAPI L3 Dependencies ---->
+
+swapi, l2, fake-swapi-l2, swapi, l3a, all, HARD, DOWNSTREAM
```

Comment:
Rather than the descriptor `fake-swapi-l2` could we use the generic `sci` descriptor? And relatedly, could we use `sci` rather than `all` for the L3A descriptor, unless `all` is a meaningful descriptor?

Reply:
I believe this will work. We will need to deploy this change to the database but otherwise, it looks good.

I think you guys know this already but to make sure we all are on same page, with your current change, it will work if you upload intermediate L2 file with filename, `imap_swapi_l2_fake-swapi-l2_v001.cdf`. Then it will kickoff all `l3a` products processing if that's what you intended. Once you test the mechanism of this dependency files, the actual processing will need to have something like this like @maxinelasp was saying. You can open another PR once you and SWAPI have defined those names.

```suggestion
swapi, l2, sci, swapi, l3a, all, HARD, DOWNSTREAM
```

or

```suggestion
swapi, l2, sci, swapi, l3a, <product-1-name>, HARD, DOWNSTREAM
swapi, l2, sci, swapi, l3a, <product-2-name>, HARD, DOWNSTREAM
swapi, l3a, <product-1-name>, swapi, l3b, <product-1-name>, HARD, DOWNSTREAM
...
```